releaseNotes/AzureDevOpsPipelineExtension.md

```diff
@@ -4,6 +4,9 @@ The Azure App Configuration extension in Azure DevOps pipeline has been publishe
 
 ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v5.2.0 - March, 02 2022
+Fixed a bug that caused an invalid_client error when using certificate authentication [#608](https://github.com/Azure/AppConfiguration/issues/608).
```

Comment:
If it's not always the case, we may say "... bug that could cause ...". Otherwise, it's good.

certificate authentication -> certificate-based authentication